easylist/easylist_general_hide.txt

```diff
@@ -15898,6 +15898,7 @@
 ##.wh-advert
 ##.wh_ad
 ##.wh_ad_inner
+##.when-show-ads
```

Comment:
Generic for one site only?

Reply:
This is EL's preference - generic if selector looks to be unique to ad.